### ISSUE TITLE
Add an argument to `basisSet` to exclude a variable or set of variables

### ISSUE THREAD

**Issue by jslefche** [jslefche/piecewiseSEM#127]

**Comment by jebyrnes** [comment]
Yes, this is part of a problem with `variable ~ 1` such that if you want to include a variable in the basis set and not have it correlate with your exogenous variables, right now it's not possible. A problem for model comparison.
